<launch>
 <rosparam ns="/bioloid/joints">
  name: ['r_shoulder_swing_joint', 'r_shoulder_lateral_joint', 'r_elbow_joint', 'l_shoulder_swing_joint', 'l_shoulder_lateral_joint', 'l_elbow_joint', 'l_hip_twist_joint', 'l_hip_lateral_joint', 'l_hip_swing_joint', 'l_knee_joint', 'l_ankle_swing_joint', 'l_ankle_lateral_joint', 'r_hip_twist_joint', 'r_hip_lateral_joint', 'r_hip_swing_joint', 'r_knee_joint', 'r_ankle_swing_joint', 'r_ankle_lateral_joint']
  servo_number: [1, 3, 5, 2, 4, 6, 8, 10, 12, 14, 16, 18, 7, 9, 11, 13, 15, 17]
  joint_encoder_offset: [512, 512, 512, 512, 512, 512, 657, 512, 512, 512, 512, 512, 351, 512, 512, 512, 512, 512]
  angle_max: [2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0]
  angle_min: [-2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0, -2.0]
  sim: false
  torque: true
 </rosparam>
</launch>
